docs-ref-conceptual/azure-cli-sp-tutorial-3.md

```diff
@@ -102,8 +102,6 @@ az keyvault secret download --file /path/to/cert.pfx \
                             --name CertName \
                             --encoding base64
 openssl pkcs12 -in cert.pfx -passin pass: -passout pass: -out cert.pem -nodes
-
-az login --service-principal -u "<myAppClientID>" -p cert.pem --tenant "<myTenantID>"
```

Comment:
No need to mention `az login` command here. It is later mentioned in "Sign in with a service principal using a certificate" section.